docs/fundamentals/documentarray/embedding.md

```diff
@@ -2,25 +2,22 @@
 # Embed via Neural Network
 
 When DocumentArray `.tensors` being set,
-you can use a neural network to {meth}`~docarray.array.mixins.embed.EmbedMixin.embed` it into it's vector representations,
-i.e. filling `.embeddings`.
-
+you can use a neural network to create vector embeddings.
 Embeddings can be used to measure the relatedness of your data.
-Embeddings are commonly used for *neural search*, *recommendation*, *clustering*, *outlier detection*, *deduplication* etc.
+Common cases include neural search, recommendation, clustering, outlier detection, deduplication etc.
 
-Docarray provides an easy interface which allows you to encode your data with
+DocArray gives you an easy interface to encode your data with
 the {meth}`~docarray.array.mixins.embed.EmbedMixin.embed` method.
-
 After calling {meth}`~docarray.array.mixins.embed.EmbedMixin.embed`,
-you should expect vector representation of your data has been stored in `.embeddings` endpoint.
+the vector representation of your data will be stored in the `.embeddings` attribute.
```

Comment:
```suggestion
the vector representation of your data will be stored in the DocumentArray's `.embeddings` attribute.
```

Reply:
it is an out dated one, maybe you should refresh?